packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter
-version: 2.0.0-nullsafety
+version: 2.0.1-nullsafety
```

Comment:
Thank you for your contribution.

The correct version : `2.0.0-nullsafety.1`

Reply:
oh ok thank you, let me change that.